grid-cli/cmd/deploy_kubernetes.go

```diff
@@ -210,7 +210,7 @@ var deployKubernetesCmd = &cobra.Command{
 			}
 		} else {
 			for i := 0; i < workerNumber; i++ {
-				workers[i].Node = workersNode
+				workers[i].Node = uint32(workersNode[i%len(workersNode)])
```

Comment:
else the workersNodes should be more than or equal the workers nodes, no? so no ned for the `i%len(workersNode)` part